eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -42,10 +43,22 @@ steps:
       displayName: Publish non-helix TestResults
       condition: always()
 
+    - ${{ if ne(parameters.isWindows, 'true') }}:
+      # E2E tests are disabled on windows for now
+      - script: ${{ parameters.dotnetScript }}
+                build
+                -restore
+                -p:ContinuousIntegrationBuild=true
+                -p:Configuration=${{ parameters.buildConfig }}
+                -t:InstallWorkloadUsingArtifacts
+                -bl:${{ parameters.repoLogPath }}/WorkloadInstallForTesting.binlog
+                $(Build.SourcesDirectory)/tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj
```

Comment:
Is it weird to pick the EndToEnd tests to have this "InstallWorkloadUsingArtifacts" target? It seems like something that should be coming from an "infrastructure" project. Not one of the test projects.

Reply:
Yes, I was thinking of fixing it in a follow up though. But I can try to do it here too.